src/huggingface_hub/utils/tqdm.py

```diff
@@ -131,6 +131,14 @@ def __init__(self, *args, **kwargs):
             kwargs["disable"] = True
         super().__init__(*args, **kwargs)
 
+    def __delattr__(self, attr):
+        """fix for https://github.com/huggingface/huggingface_hub/issues/1603"""
+        try:
+            del self.__dict__[attr]
+        except KeyError:
+            if attr != "_lock":
+                raise AttributeError(attr)
```

Comment:
I'd prefer to use the "real" underlying `__delattr__` and raise the default exception generated by it instead of creating a new `AttributeError(attr)` that is less verbose.

```suggestion
    def __delattr__(self, attr: str) -> None:
        """Fix for https://github.com/huggingface/huggingface_hub/issues/1603"""
        try:
            super().__delattr__(attr)
        except AttributeError:
            if attr != "_lock":
                raise
```